### ISSUE TITLE
Accessibility Checklists: Accessibility test pages structural improvements

### ISSUE THREAD

**Issue by amyleadem** [uswds/uswds-site#2548]
> [!Note]
> This is a master issue to track the low-effort code enhancements for the accessibility tests pages. All work will be completed in separate issues, added to the list below.

## Tasks
Address the low-effort tasks identified in #2438: 

1. Create accessibility-test layout
    - Related issue: #2503
    - Related PR: #2502
1. Rename accesibility.md checklist files to avoid conflicts & improve clarity
    - Related issue: #2579
    - Related PR: #2581
1. ~Move _data/accessibility-tests YML data into the accessibility tests markdown files~
    - ~Related issue: #2582~
    - Placing this on hold because the data needs to be shared across the main component page and the component a11y tests page. A shared file in the `_data` directory is probably best practice here.
1. Spell out numbers <10 in the test status section
    - Related issue: #2596 
    - Related PR: #2597
    - This will allow the copy to comply with content standards 
1. Fix the layouts to make sure all component pages with a related accessibility test page have:
    - Related issue: #2593 
    - Table page does not have a summary of the a11y test results
    - Typography page does not have a summary of the a11y test results or the summary of a11y test results
1. Remove header from `checklist-feedback.html`. Avoid heading order  a11y issues
1. Validate the checklist changelog fields. Ex: Is title field used? Is `type: component`accurate?

**Comment by annepetersen** [comment]
[The USWDS Open Source Community](https://github.com/uswds/uswds/blob/develop/COMMUNITY.md#communitymd) is welcome to pick this back up, but given limited federal staff bandwidth (hi, hello 👋 it only me full-time right now), I'm closing this set of issues.
